modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/objectbuilder/ObjectBuilderUtils.java

```diff
@@ -26,7 +26,7 @@
    */
   public static <T> T createInstance(Class<T> prototypeClass) {
     try {
-      return withContextClassLoader(prototypeClass.getClassLoader(), () -> prototypeClass.newInstance());
```

Comment:
this method is static and public. Not safe to predict where it's been called from